docs/node-configuration.md

```diff
@@ -216,6 +217,50 @@ where:
   setting depends on the NeoFS block storage configuration and is applicable only if
   `SkipIndexFilesSearch` is set to `false`. It's set to 128000 by default.
 
+### NeoFS StateFetcher Configuration
+
+`NeoFSStateFetcher` configuration section contains settings for NeoFS
+StateFetcher module and has the following structure:
+```
+  NeoFSStateFetcher:
+    Enabled: true
+    UnlockWallet:
+      Path: "./wallet.json"
+      Password: "pass"
+    Addresses:
+      - st1.storage.fs.neo.org:8080
+      - st2.storage.fs.neo.org:8080
+      - st3.storage.fs.neo.org:8080
+      - st4.storage.fs.neo.org:8080
+    Timeout: 10m
+    ContainerID: "7a1cn9LNmAcHjESKWxRGG7RSZ55YHJF6z2xDLTCuTZ6c"
+    StateAttribute: "State"
+    KeyValueBatchSize: 1000
+```
+where:
+- `Enabled` enables NeoFS StateFetcher module.
+- `UnlockWallet` contains wallet settings to retrieve account to sign requests to
+  NeoFS. Without this setting, the module will use randomly generated private key.
+  For configuration details see [Unlock Wallet Configuration](#Unlock-Wallet-Configuration).
+- `Addresses` is a list of NeoFS storage nodes addresses. If not specified, the
+  `Addresses` will be taken from the `NeoFSBlockFetcher` configuration section.
+- `Timeout` is a timeout for a single request to NeoFS storage node (10 minutes by
+  default).
+- `ContainerID` is a container ID to fetch blocks from. If not specified, the
+  container ID will be taken from the `NeoFSBlockFetcher` configuration section.
+- `StateAttribute` is an attribute name of NeoFS object that contains state
+  data. It's set to `State` by default.
+- `KeyValueBatchSize` is the number of key value pair to be processed and flushed 
+  to the storage. By default, it's set to 1000.
+
+
+Also `StateSyncInterval` specifies the block height interval between consecutive state
```

Comment:
`StateSyncInterval` protocol configuration value specifies the block height interval between consecutive state...